### PR TITLE
Added Xcode 9.3 beta (9Q98q)

### PR DESCRIPTION
**Pull request by Titokhan**

### CODE REVIEW

File: xcode.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ Got access to old versions of Apple installers you know are legit? Submit some h
 
 | Version                           | SHA1 Checksum                              | Filename                          |
 | --------------------------------- | ------------------------------------------ | --------------------------------- |
+| [Xcode 9.3 beta (10.12-10.13)][9.3b1] (9Q98q) | `3d1f4918810b173ff205b64a2006bb163d9cf5b1` | Xcode_9.3_beta.xip <!-- a3cf8fecf9c5a664cc8039914b51a4962546f1207e2c1514f219358167a03f4f -->
```
Comment (markmentovai) [inline xcode.md]: It’s only 10.13 now.

Comment (Titokhan) [reply xcode.md]: Aah... my bad. Should I correct it and send you another pull request?

Comment (markmentovai) [reply xcode.md]: Yeah, just push another commit with the fix to your branch, and we should be able to just do it from this pull request. I’ll merge it once it looks OK.

Comment (Titokhan) [reply xcode.md]: Well, I've done it - take a look.

File: xcode.md [hunk 2]
```diff
@@ -159,8 +159,10 @@ Got access to old versions of Apple installers you know are legit? Submit some h
 
 | SHA1 | Version | Filename |
 | ------- | ---- | -------- |
+| `f1aeb8cc3cc0092e3fcd569f1bd9234d9c25d666` | [CLI Tools 9.3 beta (10.13)][cli-9.3b1-10.13] | Command_Line_Tools_macOS_10.13_for_Xcode_9.3_beta.dmg
```
Comment (markmentovai) [inline xcode.md]: If you’re going to do this, please include a  SHA-256 for any new hashes too, as you did for the 9.3 beta above.

Comment (Titokhan) [reply xcode.md]: Not an issue... more hashes coming soon!

File: xcode.md [hunk 3]
```diff
@@ -33,6 +33,7 @@ Got access to old versions of Apple installers you know are legit? Submit some h
 
 | Version                           | SHA1 Checksum                              | Filename                          |
 | --------------------------------- | ------------------------------------------ | --------------------------------- |
+| [Xcode 9.3 beta (10.13)][9.3b1] (9Q98q) | `3d1f4918810b173ff205b64a2006bb163d9cf5b1` | Xcode_9.3_beta.xip <!-- a3cf8fecf9c5a664cc8039914b51a4962546f1207e2c1514f219358167a03f4f -->
```
Comment (Coeur) [inline xcode.md]: +1 I got the same shasum.

**Comment by markmentovai** [comment]
I’ll just commit this and add the SHA-256 myself.

**Pull request merged**